2_phase_minting_schedule/main.tex

```diff
@@ -70,74 +71,103 @@ \section{Empirical analysis of existing networks}
+
+Where $l$ is the size of the sub-stake, $L$ is the sum of all locked sub-stakes, and $S_{P1}$ is the maximum number of tokens to be minted in the first phase (approximately 1.83 billion tokens). In phase two, a similar equation can be utilized: 
+
+\begin{equation}
+    \label{eq:rate-max}
+    ds_{i,p} = \kappa \cdot \frac{l_i}{L} \cdot \frac{\ln{2}}{T_{1/2}} \cdot \left(S_{P2} - S_{p-1} \right)\\
+\end{equation}
+
+Where $S_{P2}$ is the maximum number of tokens to be minted in the second phase (approximately 1.056 billion tokens). 
 \\\\
-It's important to note that this distinction between the inflation rate and real growth rate of the supply is not a unique feature of the the two-phase proposal. In the previous draft model, the real growth rate of the supply decreases \textit{more rapidly} than the inflation rate – since it is a function of both the inflation rate's decay \textit{and} the supply's growth over time. This is particularly relevant because the second phase of the two-phase model follows a similar formula. The minting schedule after period 1825 (the last day of year 5) involves a rate which decays period-to-period, applied daily to the constant of the genesis supply (1 billion tokens). For example, 1,001,558 new tokens are minted on day 1827, which is 951 fewer than the previous day.
+The circulating supply at a given period is the sum of the tokens received thus far by all sub-stakes: 
+\begin{equation}
+    dS_p= \sum_i ds_{i,p},
+\end{equation}
+
+The implication is that it will likely take longer than 5 years to mint all the tokens reserved for phase one. The year in which the protocol switches from the first to second phase, $T_{1\rightarrow2}$, is therefore : 
+
+\begin{equation}
+    T_{1\rightarrow2}= \frac{S_{P1}}{S_0 \cdot \kappa^* \cdot I_{stable}}
+\end{equation}
+
+Where $\kappa^*$ is the average lock duration of all sub-stakes, weighted by relative size, across all periods in question. For example, if $\kappa^*$ is 73 days, then the minting schedule will switch phases about 8 1/3 years after network launch. If $\kappa^*$ is 11 months, $T_{1\rightarrow2}$ equals 5 years \& 79 days. 
 \\\\
-With this new token minting schedule, we can work out that approximately 1.83 billion tokens are minted in the first 5 years, followed by approximately a further 1.056 billion between the fifth year and t = $\infty$.
+For reference, the supply in the second phase can be calculated with this equation: 
 
-\end{document}
+\begin{equation}
+    S(t) = S_0 \left[1 + \frac{I_{stable} \cdot T_{1/2}}{\ln{2}}\left(1 - 2^{-\frac{t \cdot \kappa^*}{T_{1/2}}} \right) \right].
+\end{equation}
 
-In other words, the same amount of tokens will be emitted from $t=5$ until $t=\infty$ 
+Which implies that the rate of issuance decay is inversely proportional to $\kappa^*$. In a similar fashion to phase one, shorter average lock-ups slow down the minting of tokens. 
```

Comment:
Is the inverse proportional relationship of "rate" & "kappa" correct here? We should be careful using the word "rate" here.
If kappa is low, i.e. short durations, then the decay rate slows i.e. lowers (and vice versa)

Do you instead mean that the **half-life** (i.e. the amount of time for the quantity to decrease by half) is inversely proportional to kappa?

Reply:
Well-spotted! It's proportional, and I agree about 'rate' (the term that shall not be named). 

The reason I didnt say that kappa* is inversely proportional to the 'half-life' is that we previously say the half-life _is T_1/2 _. Kappa doesn't affect T_1/2  – as you can see from eq. 13, it modifies the expression for decay over time as a whole: -2^(-t * k / T_1/2) 

```suggestion
Which implies that issuance decay is proportional to $\kappa^*$. In a similar fashion to phase one, shorter average lock-ups slow down the minting of tokens. 
```

Reply:
👍 - Yes decay and not T_1/2 - I misread "8 1/3 years" and "5 years \& 79 days".